tests/Predis/Command/StringBitFieldTest.php

```diff
@@ -140,12 +140,11 @@ public function testBitfieldWithIncrbyModifier()
     /**
      * @group connected
      * @requiresRedisVersion >= 3.2.0
-     * @expectedException \Predis\Response\ServerException
-     * @expectedExceptionMessage Operation against a key holding the wrong kind of value
      */
     public function testThrowsExceptionOnWrongType()
     {
-        $this->markTestSkipped('Currently skipped due issues in Redis (see antirez/redis#3259).');
```

Comment:
Good. This was just leftover for a long-time fixed bug in Redis, in this case I think it makes sense to remove the `markTestSkipped()` condition while we are at this 👍